server/src/main/java/com/lunchtime/controllers/FeedbackController.java

```diff
@@ -0,0 +1,29 @@
+package com.lunchtime.controllers;
+
+import java.util.List;
+import com.lunchtime.models.Feedback;
+import com.lunchtime.service.FeedbackService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+
+@RestController
+public class FeedbackController {
+
+  private final FeedbackService feedbackService;
+
+  public FeedbackController(FeedbackService feedbackService) {
+    this.feedbackService = feedbackService;
+  }
+
+
+  @GetMapping("/api/feedback/{id}")
+  public ResponseEntity<List<Feedback>> getById(@PathVariable("id") Long id) {
+
+    List<Feedback> page = feedbackService.findByRestIdOrderByDateDesc(id);
```

Comment:
I guess `findByRestIdOrderByDateDesc()` method is your created one. If true, separate it to findRestaurantById() and way of ordering in different places.

If we need findByRestIdOrderByDate**Asc**(), so we need another almost same method.

Reply:
its JPA 
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.repositories